test/stress.cxx

```diff
@@ -1213,7 +1216,11 @@ void stress10()
       snprintf(filename,20,"Event_%d.root",file);
       chfile[file] = new TFile(filename,"recreate");
       if (file>=5) {
+#ifdef R__HAS_LZ4
+         chfile[file]->SetCompressionAlgorithm(ROOT::kLZ4);
+#else
          chfile[file]->SetCompressionAlgorithm(ROOT::kLZMA);
+#endif
```

Comment:
This may be here on purpose to test LZMA, no? Why change to LZ4? Is it just for testing?

Reply:
Thanks for good point! it was for testing and I will cleanup!